scripts/run-notebook.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This bash script is used to run the example notebooks
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+NOTEBOOK_INPUT=""
+NOTEBOOK_OUTPUT=""
+PAPERMILL_PARAMS=()
+PAPERMILL_PARAM_YAML=""
+TRAINING_PYTHON_SDK="git+https://github.com/kubeflow/training-operator.git#subdirectory=sdk/python"
+
+usage() {
+  echo "Usage: $0 -i <input_notebook> -o <output_notebook> [-p \"<param> <value>\"...] [-y <params.yaml>]"
+  echo "Options:"
+  echo "  -i  Input notebook (required)"
+  echo "  -o  Output notebook (required)"
+  echo "  -p  Papermill parameters (optional), pass param name and value pair (in quotes whitespace separated)"
+  echo "  -y  Papermill parameters YAML file (optional)"
+  echo "  -k  Kubeflow Training Operator Python SDK (optional)"
+  echo "  -h  Show this help message"
+  echo "NOTE: papermill, jupyter and ipykernel are required Python dependencies to run Notebooks"
+  exit 1
+}
+
+while getopts "i:o:y:p:k:r:d:h:" opt; do
+  case "$opt" in
+    i) NOTEBOOK_INPUT="$OPTARG" ;;            # -i for notebook input path
+    o) NOTEBOOK_OUTPUT="$OPTARG" ;;           # -o for notebook output path
+    p) PAPERMILL_PARAMS+=("$OPTARG") ;;       # -p for papermill parameters
```

Comment:
Since you name other papermill parameter as `-k`, should we name the namespace parameter as `-n`?

Reply:
@saileshd1402 Please can you check it, so we can merge the PR ?

Reply:
I added it already I think. Can you check the latest commits once?

Reply:
I think, you should also rename the papermill command here to use `-n` and `-k` flag: https://github.com/kubeflow/training-operator/blob/d3e903154604818513f29ea1e6264f3c05c8590c/scripts/run-notebook.sh#L59
Also, you can remove `-p` parameter since it is no longer needed.

Reply:
E.g. I mean this part: https://github.com/kubeflow/training-operator/blob/d3e903154604818513f29ea1e6264f3c05c8590c/scripts/run-notebook.sh#L61-L63

Reply:
Oh understood, you are saying let's remove custom parameters of papermill in this script. I think we may use it in the future but I guess we can add if and when necessary. I'll remove it for this PR

Reply:
Yeah, let's add them in the future once we need them.